### PR TITLE
Add group options

### PR DESCRIPTION
**Pull request by justinkillen**
For https://github.com/xanzy/go-gitlab/issues/762

### CODE REVIEW

File: groups.go [hunk 1]
```diff
@@ -125,13 +125,17 @@ func (s *GroupsService) GetGroup(gid interface{}, options ...OptionFunc) (*Group
...
+	Path                  *string          `url:"path,omitempty" json:"path,omitempty"`
+	Description           *string          `url:"description,omitempty" json:"description,omitempty"`
+	Visibility            *VisibilityValue `url:"visibility,omitempty" json:"visibility,omitempty"`
+    RequireTwoFactorAuth  *bool            `url:"require_two_factor_authentication,omitempty" json:"require_two_factor_authentication,omitempty"`
+    TwoFactorGracePeriod  *int             `url:"two_factor_grace_period,omitempty" json:"two_factor_grace_period,omitempty"`
+    ProjectCreationLevel  *string          `url:"project_creation_level,omitempty" json:"project_creation_level,omitempty"`
+    SubGroupCreationLevel *string          `url:"subgroup_creation_level,omitempty" json:"subgroup_creation_level,omitempty"`
+	LFSEnabled            *bool            `url:"lfs_enabled,omitempty" json:"lfs_enabled,omitempty"`
+	RequestAccessEnabled  *bool            `url:"request_access_enabled,omitempty" json:"request_access_enabled,omitempty"`
+	ParentID              *int             `url:"parent_id,omitempty" json:"parent_id,omitempty"`
```
Comment (svanharmelen) [inline groups.go]: Please use `go fmt` to make sure the code is formatted properly.

File: groups.go [hunk 2]
```diff
@@ -125,13 +125,17 @@ func (s *GroupsService) GetGroup(gid interface{}, options ...OptionFunc) (*Group
...
-	RequestAccessEnabled *bool            `url:"request_access_enabled,omitempty" json:"request_access_enabled,omitempty"`
-	ParentID             *int             `url:"parent_id,omitempty" json:"parent_id,omitempty"`
+	Name                  *string          `url:"name,omitempty" json:"name,omitempty"`
+	Path                  *string          `url:"path,omitempty" json:"path,omitempty"`
+	Description           *string          `url:"description,omitempty" json:"description,omitempty"`
+	Visibility            *VisibilityValue `url:"visibility,omitempty" json:"visibility,omitempty"`
+    RequireTwoFactorAuth  *bool            `url:"require_two_factor_authentication,omitempty" json:"require_two_factor_authentication,omitempty"`
+    TwoFactorGracePeriod  *int             `url:"two_factor_grace_period,omitempty" json:"two_factor_grace_period,omitempty"`
+    ProjectCreationLevel  *string          `url:"project_creation_level,omitempty" json:"project_creation_level,omitempty"`
+    SubGroupCreationLevel *string          `url:"subgroup_creation_level,omitempty" json:"subgroup_creation_level,omitempty"`
```
Comment (svanharmelen) [inline groups.go]: According to the docs this should be an `*int`, but I wonder if that could be wrong in the docs?

Comment (justinkillen) [reply groups.go]: Yes, the field type is wrong in the table - it's clearly a string from the description and the example responses.  I created an issue for the gitlab docs team: https://gitlab.com/gitlab-org/gitlab-docs/issues/642

File: groups.go (hunk 1 above)
Comment (justinkillen) [reply groups.go]: will do

**Comment by justinkillen** [comment]
Added all fields from the CreateGroup API

**Pull request merged**